framework/src/play/jobs/Job.java

```diff
@@ -164,7 +164,19 @@ public void every(String delay) {
      *            time in seconds
      */
     public void every(int seconds) {
-        JobsPlugin.executor.scheduleWithFixedDelay(this, seconds, seconds, TimeUnit.SECONDS);
+        // Wrap this job in a Runnable that catches and ignores all thrown exception.
+        // If there's an unhandled exception, then the executor would suppress all
+        // further executions, violating the contract.  Note that this does
+        // not interfere with the normal handling of onException().
+        Runnable neverThrowingRunnable = new Runnable() {
```

Comment:
I also dislike anonymous class wrappers.  The reason I didn't add a new flag to the Job class is that it's not a property of the job, but how the job was scheduled.  I didn't want to introduce hard-to-understand failures when a Job is scheduled multiple times by the application.  For example, if someone wanted to run a Job now and every 24 hours afterward, they could write something like:

```java
  Job j = MyJob();
  j.every("24h");
  try {
    await(j.now());
  } catch (Exception ex) {
    error("could not run the job");
  }
```

If `Job.every()` sets a flag in the Job object, then putting `j.every()` before the `j.now()` would interfere with the exception propagation, but the reverse wouldn't.  To me, that seemed counter-intuitive.  The current Job implementation uses the anonymous wrapper class created in `Job.getJobCallingCallable()` to manage the exceptions for jobs scheduled by `Job.now()` and `Job.in()` without reading/writing state in the Job, so I followed suit.

That said, the applications written by my organization never the above pattern, so we wouldn't be negatively impacted by keeping a "scheduled by every" flag within the Job object.  So if that's the way you want me to go, that's the way I'll go.  Please confirm.

I suppose you recommended making this behavior controllable by a new configuration option so that someone could disable it if they _wanted_ an unhandled exception to halt the "every" execution.  Are there additional documentation requirements for introducing a new configuration option, or would it be acceptable to keep it as a hidden option, available to anyone who reads the source code?